src/components/Map.vue

```diff
@@ -50,7 +56,36 @@ export default {
         return [[bbox[1], bbox[2]], [bbox[3], bbox[0]]];
       }
       return null;
-    }
+    },
+    getBaseMap() {
+      if ('summaries' in this.stac) {
+        if ('ssys:targets' in this.stac.summaries){
+          var target = this.stac.summaries['ssys:targets'][0]
+          switch (target.normalize()) {
```

Comment:
Are you sure you really want to use "normalize" here? Feels more like a case for toLowerCase.